### ISSUE TITLE
[studio] Studio GET APIs with long query string returns error.

### ISSUE THREAD

**Issue by yacdaniel** [craftercms/craftercms#4497]
## Describe the bug
When we send long query string in Studio GET API's, like `get_delete_package`  an error is thrown because of a big request header. 

## Tasks
- [x] Identify the affected APIs (@dejan-brkic can help)
- [x] Update docs
- [x] Update backend code
- [x] Update UI
- [ ] Test

## To Reproduce
1. Create a site from videocenter bp.
2. Go to dashboard and click `Show In-Progress Items`
3. Select all items and click `Delete`
4. See error in console, dialog will not load. 

## Expected behavior
Request completes without error. 

## Screenshots
N/A

## Logs
```
<!doctype html><html lang="es"><head><title>Estado HTTP 400 – Bad Request</title><style type="text/css">body {font-family:Tahoma,Arial,sans-serif;} h1, h2, h3, b {color:white;background-color:#525D76;} h1 {font-size:22px;} h2 {font-size:16px;} h3 {font-size:14px;} p {font-size:12px;} a {color:black;} .line {height:1px;background-color:#525D76;border:none;}</style></head><body><h1>Estado HTTP 400 – Bad Request</h1><hr class="line" /><p><b>Tipo</b> Informe de Excepción</p><p><b>mensaje</b> La cabecera del requerimiento es demasido grande</p><p><b>Descripción</b> El requerimiento enviado por el cliente era sintácticamente incorrecto.</p><p><b>excepción</b></p><pre>java.lang.IllegalArgumentException: La cabecera del requerimiento es demasido grande
	org.apache.coyote.http11.Http11InputBuffer.fill(Http11InputBuffer.java:770)
	org.apache.coyote.http11.Http11InputBuffer.parseRequestLine(Http11InputBuffer.java:466)
	org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:502)
	org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:818)
	org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1626)
	org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	java.base&#47;java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	java.base&#47;java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	java.base&#47;java.lang.Thread.run(Thread.java:834)
</pre><p><b>nota</b> La traza completa de la causa de este error se encuentra en los archivos de registro del servidor.</p><hr class="line" /><h3>Crafter CMS</h3></body></html>
```
### Version
`4.0.x` & `3.1.x`

### OS
Any

### Browser
Any

## Additional context
N/A

**Comment by jdrossl** [comment]
@sumerjabri there are two options to fix this:
1. Change the API to use POST (the size limit is greater than for GET)
2. Change the configuration of Tomcat to increase the limit

**Comment by sumerjabri** [comment]
#1, please.

**Comment by jdrossl** [comment]
@rart the need to be updated too

**Comment by rart** [comment]
Please provide full list of apis that changed when done.

**Comment by jdrossl** [comment]
@rart Here is the list of endpoints that changed:
```
DELETE ​/content​/delete
GET ​/content​/children_by_id
GET ​/content​/children_by_path
GET​ /content​/sandbox_items_by_id
GET ​/content​/sandbox_items_by_path
GET ​/dependency​/dependencies
GET ​/translation​/item_by_id
GET ​/translation​/item_by_path
```

**Comment by yacdaniel** [comment]
Verified fixed in `4.0.0-SNAPSHOT`
